docs/development.md

```diff
@@ -28,7 +28,32 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 
 * To stop the dev server, click the "Terminate" icon on the Eclipse console.
 
-### Outside Eclipse
+### With IntelliJ
+
+> If this is your first time urnning the dev server, you will need to set up the required `Run Configuration`.
+
+#### Set up the Run Configuration
+
+1. In your `Event Log`, you should see this line: `Frameworks detected: Google App Engine, Web, JPA frameworks are detected in the project`. Click `Configure` and `OK` in the dialog box that appears.
+1. Go to `File → Project Structure...`.
+1. Under `Artifacts → Gradle : teammates.war (exploded)`, check `Include in project build`.
+1. Click `OK`.
+1. Got to `Run → Edit Configurations...`.
+1. Click `+ → Google AppEngine Dev Server`.
+1. Name it `Dev Server`.
+1. Click `Configure` next to `Application server`.
+1. Click `+ → ...`. Select the App Engine SDK you downloaded in Step 3 of the `Setting up a development environment` guide.
```

Comment:
Can consider replacing the `Setting up a development environment` with a link to the setup guide